apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/SimpleSubscriberTest.cs

```diff
@@ -142,11 +144,16 @@ public void Dispose()
 
             private class PullStream : StreamingPullStream
             {
-                public PullStream(IEnumerable<ServerAction> msgs, List<TimedId> extends, List<TimedId> acks, List<DateTime> writeCompletes,
+                public PullStream(TimeSpan writeAsyncPreDelay,
+                    IEnumerable<ServerAction> msgs, List<TimedId> extends, List<TimedId> acks, List<DateTime> writeCompletes,
                     IScheduler scheduler, IClock clock, TaskHelper taskHelper, CancellationToken? ct) =>
-                    (_en, _clock, _extends, _acks, _writeCompletes) = (new En(msgs, scheduler, taskHelper, ct), clock, extends, acks, writeCompletes);
+                    (_taskHelper, _scheduler, _writeAsyncPreDelay, _en, _clock, _extends, _acks, _writeCompletes) =
+                        (taskHelper, scheduler, writeAsyncPreDelay, new En(msgs, scheduler, taskHelper, clock, ct), clock, extends, acks, writeCompletes);
```

Comment:
I think the time has come to break this into multiple statements. At the moment it's just a noise of commas and identifiers.

Reply:
Done :)